### PR TITLE
PMM-5192 Replace go1.12 with go1.13.

### PR DESCRIPTION
**Pull request by artemgavrilov**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-675-c67ad9d
client docker - perconalab/pmm-client-fb:PR-675-c67ad9d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-675-c67ad9d.tar.gz

File: build/rpmbuild-docker/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ RUN yum install -y --setopt=skip_missing_names_on_install=False \
 RUN yum -y remove nodesource-release-el7-1.noarch
 RUN yum clean all && rm -rf /var/cache/yum
 
-ENV GO_TARBALL https://dl.google.com/go/go1.12.12.linux-amd64.tar.gz
+ENV GO_TARBALL https://dl.google.com/go/go1.13.7.linux-amd64.tar.gz
```
Comment (AlekSi) [inline build/rpmbuild-docker/Dockerfile]: We need that change.

Comment (artemgavrilov) [reply build/rpmbuild-docker/Dockerfile]: Done

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-675-aee4fb8
client docker - perconalab/pmm-client-fb:PR-675-aee4fb8
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-675-aee4fb8.tar.gz

File: build/rpmbuild-docker/Dockerfile (hunk 1 above)
Comment (artemgavrilov) [reply build/rpmbuild-docker/Dockerfile]: Done

**Comment by AlekSi** [comment]
@EvgeniyPatlan will you rebuild and upload the container?

**Comment by EvgeniyPatlan** [comment]
@AlekSi please ask @Sevlin :)

File: build/rpmbuild-docker/Dockerfile [hunk 2]
```diff
@@ -19,8 +19,12 @@ RUN yum install -y --setopt=skip_missing_names_on_install=False \
 RUN yum -y remove nodesource-release-el7-1.noarch
 RUN yum clean all && rm -rf /var/cache/yum
 
-ENV GO_TARBALL https://dl.google.com/go/go1.12.12.linux-amd64.tar.gz
-RUN wget --progress=dot:giga ${GO_TARBALL} -O /tmp/golang.tar.gz
+ENV GO_VERSION 1.13.7
```
Comment (AlekSi) [inline build/rpmbuild-docker/Dockerfile]: I would keep either `GO_TARBALL` or that one, not both.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-675-f1a15b6
client docker - perconalab/pmm-client-fb:PR-675-f1a15b6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-675-f1a15b6.tar.gz

File: build/rpmbuild-docker/Dockerfile (hunk 2 above)
Comment (artemgavrilov) [reply build/rpmbuild-docker/Dockerfile]: Make sense, done.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-675-441734f
client docker - perconalab/pmm-client-fb:PR-675-441734f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-675-441734f.tar.gz

**Pull request merged**